tests/Policy_1.py

```diff
@@ -22,11 +22,15 @@
 #
 
 import unittest
+import os.path
 
 import lsst.utils.tests
 from lsst.pex.policy import Policy, NameNotFound
 import lsst.pex.exceptions
 
+
+proddir = lsst.utils.getPackageDir('pex_policy')
```

Comment:
Given that there were already instances of `proddir` in the tests I think I will leave them as is.